clio-client/src/main/scala/org/broadinstitute/clio/client/commands/GetGvcfSchemaExecutor.scala

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.clio.client.commands
```

Comment:
This should be in package `org.broadinstitute.clio.client.dispatch` (it's a bug in the existing code for wgs-ubams, getting fixed in #104).